policy/bundle.go

```diff
@@ -966,6 +1028,10 @@ func (c *bundleCache) precompileQuery(query *explorer.Mquery, policy *Policy) *e
 		return nil
 	}
 
+	if query.Title == "" {
```

Comment:
this happens after the compile call, is this intended?

Reply:
I think this is happening before compile as the function is `precompileQuery`.

Reply:
Can you explain further. I still see a precompileQuery in cnquery